org.ekstep.contenteditorfunctions-1.2/editor/plugin.js

```diff
@@ -598,6 +599,12 @@ org.ekstep.contenteditor.basePlugin.extend({
             }
             data.callback && data.callback(err, res);
         });
+    },
+    updateHierarchyVersionKey: function(){
```

Comment:
This is not required.
Version key update can be done in saveCollectionContent() method